app/views/patients/_pledge.html.erb

```diff
@@ -35,7 +35,12 @@
       <% if current_tenant.pledge_generation_config.present? %>
         <p class="row"><%= t('patient.pledge.step_three.generated', fund: current_tenant.name) %></p>
         <p class="row">
-          <%= t('patient.pledge.step_three.downloads') %> <%= fax_service %>
+          <%= t('patient.pledge.step_three.downloads') %>
+          <% if patient.clinic&.email_for_pledges.present? %>
+              <%= clinic_pledge_email(patient) %>
+          <% elsif fax_service.present? %>
+              <%= fax_service %>
+          <% end %>
```

Comment:
@colinxfleming is this what you had in mind? I think the logic is flipped around from what you had written in #2528

Reply:
ok, nvm change this to

- display email if email
- otherwise display fax number (if exists) and fax service